packages/sycamore-macro/src/component.rs

```diff
@@ -307,10 +372,12 @@ fn inline_props_impl(item: &mut ItemFn) -> Result<TokenStream> {
 
     let doc_comment = format!("Props for [`{}`].", item.sig.ident);
 
+    let attrs = attrs.into_iter().collect::<Vec<_>>();
```

Comment:
We could then change this to
```rust
attrs.into_iter().map(|attr| Attribute {
    pound_token: parse_quote!(#),
    // ...
}).collect::<Vec<_>>()
```
so that we generate the `#[...]` brackets around the attribute. (see previous comment)